README.md

```diff
@@ -158,10 +158,122 @@ This will determine any missing `ansible-core` tags and create them in `ansible-
 nox -s tag
 
 # If you use a different upstream remote, specify the name.
-nox -s tag -- --remote <name> tag 
+nox -s tag -- --remote <name> tag
 
 # If your core repo is not in the same filesystem location, specify the path.
 nox -s tag -- --core <path> tag
 ```
 
 See `nox -s tag -- --help` for extended options.
+
+## Branching for new stable versions
```

Comment:
Can we put this in the CONTRIBUTING.md maybe? I don't think we need a process we do twice a year in the main README.

Reply:
I'm okay with that or a separate BRANCHING_PROCESS.md document, whatever you think. I just don't want too much information that's not relevant to the average contributor in the main README.

Reply:
After reading the comment from @samccann yesterday I think it'd be good to move this to `MAINTAINERS.md` so we can capture the details about the release checklist procedures in a follow up.